lib/ExpensiMark.js

```diff
@@ -587,7 +587,8 @@ export default class ExpensiMark {
         });
 
         // Unescaping because the text is escaped in 'replace' function
-        replacedText = _.unescape(replacedText);
+        // We use 'htmlDecode' instead of 'unescape' to replace entities like '&#32;'
+        replacedText = Str.htmlDecode(replacedText);
```

Comment:
should we use htmlEncode the the replace function? Or just keep this change for the htmlToText function?

Reply:
We can, but I don't think it's necessary. The only reason `_.unescape()` doesn't work is because we're adding `&#32;` in the `replace()` function. Otherwise it would work.